include/EngineClient.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Engine.h - engine-system of LMMS
```

Comment:
Replace `Engine.h` with `EngineClient.h`. Optionally, remove or update the copyright info (this is your original work, not Toby's).